src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -114,7 +32,8 @@ Just set the `pinned` prop.
 			'app-navigation-entry--pinned': pinned,
 			'app-navigation-entry--editing' : editing,
 			'app-navigation-entry--deleted': undo,
-			'app-navigation-entry--collapsible': collapsible
+			'app-navigation-entry--collapsible': collapsible,
+			'active': isActive
```

Comment:
Will `<AppNavigationItem class="active">` still work with this? This is needed if the prop `to` is not used.

Reply:
Yes @korelstar, we're going to have both :)